bicep/main.bicepparam.example

```diff
@@ -0,0 +1,86 @@
+using 'main.bicep'
+
+// User defined
+param clientName = ''
+
+// Constants
+param logAnalyticsWorkspaceName = ''
+param tags = {
+  group: ''
+  client: clientName
+  ownerEmail: ''
+}
+param appServicePlanName  = '121'
+param runtimeStack = 'NODE|14-lts'
+param corsAllowedOrigins = [
+  'https://${clientName}.121.global'
+  'https://*.${clientName}.121.global'
+  'https://121-${clientName}.azurewebsites.net'
+]
+param dnsZoneName = '121.global'
+param cName = '121-${clientName}.azurewebsites.net'
+
+// Names
+param appServiceName = '121-${clientName}'
+param staticWebAppName = '121-portal-${clientName}'
+
+// AppService Env Variables
```

Comment:
Would it be possible to not have _yet another place_ that list all these variables?